### DISCUSSION TITLE
virtual scroll how do we do it ?

### DISCUSSION THREAD

**Discussion by neonarc4** [idiotWu/smooth-scrollbar#554] [Q&A] [answered]
i set the smooth scroll but it doenst work for virtual scroll why is that 

any suggestion will be helpful

**Comment by sadeghbarati** [comment]
Can you share your project?

**Reply by neonarc4** [reply]
i will create a playground soon in codesandbox  after 1-2hr just doing some dinner now

**Reply by neonarc4** [reply]
https://stackblitz.com/edit/quasarframework-miedne?file=src%2Fpages%2FIndexPage.vue here bro

**Reply by sadeghbarati** [reply]
Don't use `smooth-scrollbar` (Virtual Scroll) inside another Virtual Scroll (q-table) which has different API for calculation scroll offsets

**Reply by neonarc4** [reply]
i didnt understand what can i do to avoid it ? since i have global smooth scrollbar even i remove inside smooth-scrollbar1 the smoothscolbar_id0 move cursor down unable to interact with virtual scroll within in it

**Reply by sadeghbarati** [reply, marked as answer]
Quasar playground doesn't react when I used console.log or alert

Maybe this comment could help you
https://github.com/idiotWu/smooth-scrollbar/issues/92#issuecomment-324981807

```ts
const el = document.querySelector('.scroll');

[('touchmove', 'mousewheel', 'wheel')].forEach((eventType) => {
  el.addEventListener(eventType, (e) => e.stopPropagation());
});
```

**Reply by neonarc4** [reply]
nvm solved its seems library issue of charts @sadeghbarati  >.< thnx
